src/ansys/mapdl/core/common_grpc.py

```diff
@@ -85,11 +85,11 @@
 }
 
 
-class GrpcError(RuntimeError):
+class GrpcError(MapdlRuntimeError):
     """Raised when gRPC fails"""
 
     def __init__(self, msg=""):
-        RuntimeError.__init__(self, msg)
+        MapdlRuntimeError.__init__(self, msg)
```

Comment:
```suggestion
        super().__init__(self, msg)
```

Reply:
I know... I normally do that.. for some reason... past German decided all the error classes should do that.  I think he had his reasons... but I can't remember why now 😢

Reply:
Maybe because this "guarantees" that the super method has the right signature?

Reply:
very likely :)